doc/tutorial/single_cell_model.rst

```diff
@@ -1,11 +1,11 @@
 .. _tutorialsinglecell:
 
-A simple single cell model
+A simple single-cell model
```

Comment:
Isn't this contrary to the changes you made in all other instances?

Reply:
Grammarly suggested so ==